tests/Kernel/ContentRenderTest.php

```diff
@@ -115,12 +123,29 @@ protected function setUp() {
    * Tests that the News node type is rendered with the correct ECL markup.
    */
   public function testNews(): void {
+
+    $media = $this->container
+      ->get('entity_type.manager')
+      ->getStorage('media')->create([
+        'bundle' => 'av_portal_photo',
+        'oe_media_avportal_photo' => 'P-038924/00-15',
```

Comment:
This seems to work without an internet connection too, I guess we are not performing any actual call to the AV Portal service.